components/dash-table/src/dash-table/components/Table/Table.less

```diff
@@ -504,13 +516,22 @@
             th {
                 white-space: nowrap;
 
-                .column-header--clear,
-                .column-header--delete,
-                .column-header--edit,
-                .column-header--hide,
-                .column-header--sort {
-                    .not-selectable();
-                    cursor: pointer;
+                .column-header {
+                    display: flex;
+                    justify-content: space-between;
+
+                    &--clear,
+                    &--delete,
+                    &--edit,
+                    &--hide,
+                    &--sort {
+                        .not-selectable();
+                        cursor: pointer;
+                    }
+
+                    > *:only-child {
+                        margin-left: auto;
+                    }
                 }
```

Comment:
This is the only changes on this less file.. other changes are just prettier fix.